### DISCUSSION TITLE
GitHub Enterprise Cloud/SAML and mobile app

### DISCUSSION THREAD

**Discussion by blarsen** [community/community#37364] [Enterprise]
My employer has GitHub Enterprise Cloud set up with SAML login. Is it possible for me to use the GitHub iOS app to access my account there?

**Comment by ossanna16** [comment]
Hi there @blarsen and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by aconsuegra** [comment]
Hi @blarsen, thanks for getting in touch! You should be able to use the app with an Enterprise Account that is SAML protected. Could you give it a try and let us know if you see any issues? Thanks!
